docs_src/microservices/configuration/CommonConfiguration.md

```diff
@@ -1,60 +1,59 @@
 # Common Configuration
 
-The tables in each of the tabs below document configuration properties that are common to all services in the EdgeX Foundry platform. Service-specific properties can be found on the respective documentation page for each service.
+The tables in each of the tabs below document configuration properties that are common to all services in the EdgeX Foundry platform. 
+Service specific properties can be found on the respective documentation page for each service.
+
+!!! edgey - "EdgeX 3.0"
+    New in Edgex 3.0 the common configuration is now in a single location, both source file and in the common section in the Configuration Provider. In prior releases the common configuration was duplicated in each service's configuration and section in the Configuration Provider.
```

Comment:
you mention single location then indicate multiple. How about rewording it to this?
```suggestion
    New in Edgex 3.0 the common configuration is now in a single location. The source file is loaded into the common section in the Configuration Provider. In prior releases the common configuration was duplicated in each service's configuration and in each service's section in the Configuration Provider.
```